sources/packages/backend/libs/sims-db/src/entities/sfas-application.model.ts

```diff
@@ -129,4 +130,257 @@ export class SFASApplication extends BaseSFASApplicationModel {
     nullable: false,
   })
   extractedAt: Date;
+
+  /**
+   * The unique application number assigned to this application (Application_alias.application_no).
+   */
+  @Column({
+    name: "application_number",
+    nullable: true,
+  })
+  applicationNumber?: number;
+
+  /**
+   * (Spouse/dependents/parents)|To verify information provided to SDPR
+   * for the purpose of determining or auditing
+   * eligibility (application.study_period_living_at_home_flg).
+   */
+  @Column({
+    name: "living_arrangements",
+    nullable: true,
+  })
+  livingArrangements?: string;
+
+  /**
+   * Status (single, common law, married, separated, divorced).
+   * This is a code eg. MA for married (application_assessment.student_marital_status_cde).
+   */
+  @Column({
+    name: "marital_status",
+    nullable: true,
+  })
+  maritalStatus?: string;
+
+  /**
+   * Applicant's marriage date (applicant_spouse.marriage_dte).
+   */
+  @Column({
+    name: "marriage_date",
+    type: "date",
+    nullable: true,
+  })
+  marriageDate?: string;
+
+  /**
+   * Immigration status for student, BC residency (application.bc_residency_flag).
+   */
+  @Column({
+    name: "bc_residency_flag",
+    nullable: true,
+  })
+  bcResidencyFlag?: string;
+
+  /**
+   * Permanent residency status (application.permanent_resident_flg).
+   */
+  @Column({
+    name: "permanent_residency_flag",
+    nullable: true,
+  })
+  permanentResidencyFlag?: string;
+
+  /**
+   * Amount declared by the applicant on the application for gross income from previous year
+   * (applicant_income.income_amt  (applicant_income.applicant_owner_cde = ''A'' and applicant_income_.income_type_cde = ''APYI'')).
```

Comment:
We do not need to escape the single quotes for typescript comments.

Reply:
left over 😊